bridge/opentracing/test/go.mod

```diff
@@ -1,6 +1,8 @@
 module go.opentelemetry.io/otel/bridge/opentracing/test
 
-go 1.21
+go 1.21.0
+
+toolchain go1.22.5
```

Comment:
Are we fine with these changes?

I think that `go 1.21.0` is required for the bump and it should not be a problem..

However, I do not think we need to add `toolchain go1.22.5`. I think it would be better to toolchain consistently in all our Go modules.